### PR TITLE
FIX KEYERROR EXCEPTION

### PR DESCRIPTION
**Pull request by Amedeo91**
Hello,
for my monitoring dashboard, I want to define endpoints that are not Flask routes. 
Those are the only point where I have errors due to key errors exceptions.

Thanks,
Amedeo

### CODE REVIEW

File: flask_monitoringdashboard/controllers/endpoints.py
Comment (FlyingBird95) [inline flask_monitoringdashboard/controllers/endpoints.py:154]: I think this should be un-indented. We always want to add a decorator

File: flask_monitoringdashboard/controllers/endpoints.py
Comment (FlyingBird95) [inline flask_monitoringdashboard/controllers/endpoints.py:152]: this line was already in the old code, but I think that's incorrect, since we're not making database changes, so committing has no effect. Can you remove this line?

**Comment by FlyingBird95** [comment]
Hey Amedeo91,

thanks for your contributions to the Flask-MonitoringDashboard. I've left two comments. Please adjust them. I also changed your target branch to our development branch.

File: flask_monitoringdashboard/controllers/endpoints.py
Comment (Amedeo91) [reply flask_monitoringdashboard/controllers/endpoints.py:152]: Yes I will!

File: flask_monitoringdashboard/controllers/endpoints.py
Comment (Amedeo91) [reply flask_monitoringdashboard/controllers/endpoints.py:154]: Actually not quite. For my case, I want to log endpoints that are not Flask routes. Hence, I do not need to make remove and then add the decorator. Does it make sense?

**Comment by Amedeo91** [comment]
Hey FlyingBird95,

I reply to your comment and update the PR.
Thank you for your time!

**Review by FlyingBird95 — APPROVED** [review]

**Comment by Amedeo91** [comment]
The changes in this PR are available also here: https://github.com/flask-dashboard/Flask-MonitoringDashboard/pull/409
Hence, we should not merge this if we merge the one above

**Comment by FlyingBird95** [comment]
Let's merge #409 after it's ready

**Pull request closed without merging**
